test/runAll.spec.js

```diff
@@ -161,4 +161,31 @@ describe('runAll', () => {
       expect(err).toEqual('test')
     }
   })
+
+  it.only('should skip stashing changes if no lint-staged files are changed', async () => {
```

Comment:
Please remove `only` to enable all tests again

Reply:
Whoops! I've fixed this in my next commit :)